vars/createUploadTestReportManifest.groovy

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/** Library to run the report workflow to generate test-report manifest.
+ @param Map args = [:] args A map of the following parameters
+ @param args.testManifest <required> - The path of the test manifest used
+ @param args.buildManifest <required> - The path of the build manifest of OpenSearch artifacts.
+ @param args.dashboardsBuildManifest <optional> - The path of the build manifest of OpenSearch Dashboards
+ @param args.testRunID <required> - Test run id of the test workflow being reported.
+ @param args.testType <required> - Type of the test workflow being reported.
+ @param args.componentName <Optional> - Components that workflow runs on.
+ */
+
+def call(Map args = [:]) {
+    lib = library(identifier: 'jenkins@main', retriever: legacySCM(scm))
+
+    if (!parameterCheck(args.testManifest, args.buildManifest, args.testRunID, args.testType)) return
```

Comment:
What is this returning?

Reply:
It's not returning anything. It's returned from this library and skips the rest of code below.